R/deploy-site.R

```diff
@@ -114,6 +119,7 @@ deploy_to_branch <- function(pkg = ".",
                          commit_message = construct_commit_message(pkg),
                          branch = "gh-pages",
                          remote = "origin",
+                         build_cname = (branch == "gh-pages"),
```

Comment:
Do you need to parenthesis here?

Reply:
Not really, but I tried without and this felt a little less confusing